src/index.js

```diff
@@ -163,6 +165,7 @@ let validationMessages = {
   'requiredIf:$1:$2': requiredIf.message,
   'taxId:$1': taxId.message,
   'multipleOf:$1': multipleOf.message,
+  'hostname': hostname.message,
```

Comment:
CMIIW, but shouldn't the `'hostname'` be put at line 181? after `fqdn`, and it doesn't need `'`

Reply:
oops sorry i didn't realize that its ordered